### ISSUE TITLE
Quotas not working (v6.02)

### ISSUE THREAD

**Issue by adbario** [virtualmin/virtualmin-gpl#31]
Hi, with the latest Webmin and Virtualmin, quotas are not saved anymore for virtual servers when users home directory is outside of the default "/home" on separate disk partition. Virtualmin sets unlimited quotas event though default account plan has 1024 MB quota.

Webmin 1.869
Virtualmin 6.02
CentOS 7.4
Home directory base: /var/www/vhosts

```
$ findmnt -r

/var/www/vhosts /dev/vdb1 ext4 rw,relatime,seclabel,quota,usrquota,grpquota,data=ordered
```

Also bandwidth monitoring doesn't show up on System Information anymore and the alert box "Information! Virtualmin's configuration has not been checked since..." doesn't go away even after re-checking connfiguration.

All these were still working fine with v6.01 few days ago.

Edit: bandwidth came up after a while (clean install), quotas are still missing.

**Comment by adbario** [comment]
Downgrading back to v.6.01 fixes the issues: quotas are working normally again and are set properly, also Virtualmin config re-check alert disappears after running it.

**Comment by adbario** [comment]
Found the issue finally...

There's a check to have either mod_cgi or mod_cgid module available for Apache, but that check ahs an error on the error message name where it's trying to point, and this makes "Re-check configuration" to hang if neither of these modules are available.

Also very strange behaviour elsewhere after this, as the config check doesn't finish properly and seems to skip some things. There was also a typo in the actual error message.

<img width="751" alt="screen shot 2017-12-07 at 2 01 21" src="https://user-images.githubusercontent.com/22136575/33696274-645ffd44-daf9-11e7-8536-321cde1444f4.png">

Fixed here #32 

What's the reason to make Apache to use cgi? Let's say that you're running a server only for websites and no other services, and if there's no use for cgi then it shouldn't be forced.

**Comment by adbario** [comment]
Forcing cgi was added because of issue with email autoconfig scripts #28. I think there should be some kind of check to only require cgi if those email config scripts are in use?
